adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAMMain.scala

```diff
@@ -69,6 +70,17 @@ object ADAMMain extends Logging {
       )
     )
 
+  def main(args: Array[String]) {
+    new ADAMMain(defaultCommandGroups).apply(args)
```

Comment:
Injecting an optional doesn't really fit the dependency injection model.  What gets injected into the constructor shouldn't be mucked with after the fact.  Guice forbids injecting null by default.